manifests/integreatly-apicurito/7.5.0/apicuritooperator.v7.5.0.clusterserviceversion.yaml

```diff
@@ -1,173 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@philbrookes Are we doing upgrades for Apicurito? If we are, then we'd need to keep this csv and add a `replaces` field to the new one :thinking:

Reply:
yes, this should also work for an upgrade. I overlooked this in the jira as we thought it was going into 2.0.0 initially.

Reply:
Will add back in, thanks